engine/setContext.ftl

```diff
@@ -3,6 +3,8 @@
 [#-- Shared Provider Configurations --]
 [@includeSharedComponentConfiguration component="shared" /]
 [@includeSharedComponentConfiguration component="baseline" /]
+[@includeSharedComponentConfiguration component="s3" /]
+[@includeSharedComponentConfiguration component="ec2" /]
```

Comment:
Do we need to include new providers? I thought we were trying to drop this?

Reply:
Its temporary - they disappear in the districts PR.